components/NavBar/index.js

```diff
@@ -33,14 +33,15 @@ const NavBar = () => {
         })
         .then((responseJson) => {
           if (responseJson.incompleteUserDetails) {
-            return window.location.replace(
-              'https://my.realdevsquad.com/signup'
-            );
+            window.open('https://my.realdevsquad.com/signup', '_blank');
```

Comment:
Should we show a prompt to inform the user what's happening and then let them take the call? "Looks like you haven't filled in your profile information, would you like to do that?"

Reply:
We can show a prompt but then we will not be able to persuade the users to fill in their profile information